src/MediaHandler.js

```diff
@@ -357,6 +357,7 @@ export default class MediaHandler {
       headers: {
         range: 'bytes=0-8192',
         'accept-encoding': 'identity',
+        accept: 'image/jpeg,image/jpg,image/png,image/gif,video/mp4,application/xml,image/x-icon',
```

Comment:
why application/xml ? is this for svg?

Reply:
Yeah wasn't 100% sure if that is needed. Pulled it from [here](https://github.com/adobe/helix-admin/blob/6af10e82cd4de6a5d8816fb0a587b6c7f302f9d5/src/preview/media-upload.js#L133).